README.md

```diff
@@ -1,18 +1,75 @@
-# QleetOS User Documentation
+# qleetctl
 
-User docs for QleetOS.
+Install and manage instances of the QleetOS.
 
-## Local Development
+## Requirements
+* [go 1.18](https://go.dev/doc/install)
+* [curl](https://help.ubidots.com/en/articles/2165289-learn-how-to-install-run-curl-on-windows-macosx-linux)
+* [wget](https://www.gnu.org/software/wget/)
+* [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [homebrew](https://brew.sh/) MacOS users
 
-Prerequisistes:
+## Install
 
-* [mkdocs](https://www.mkdocs.org/)
+### Installation from public repository
 
-Run the server locally:
+Download pre-compiled binaries:
+set `VERSION` environment variable to latest
+```bash
+VERSION=$(curl -sL https://github.com/qleet/qleetctl/releases/ | xmllint -html -xpath '//a[contains(@href, "releases")]/text()' - 2> /dev/null | grep -P '^v' | head -n1)
+```
+or a specific version
+```bash
+VERSION=v0.0.2
+```
+then download and install binaries
+```bash
+wget https://github.com/qleet/qleetctl/releases/download/${VERSION}/qleetctl_${VERSION}_$(echo $(uname))_$(uname -m).tar.gz -O - |\
+    tar -xz && sudo mv qleetctl /usr/local/bin/qleetctl
+```
+
+### Homebrew
+Available for MacOS and Linux.
+
+```bash
+brew tap qleet/tap
+brew install qleet/tap/qleetctl
+```
 
+## Release
+Run `release` target
 ```bash
-mkdocs serve
+make release
 ```
 
-View the site at [http://127.0.0.1:8000](http://127.0.0.1:8000/)
+## Quickstart
 
+Install the QleetOS control plane:
+
+```bash
+qleetctl install
+```
+
+Remove the QleetOS control plane:
+
+```bash
+qleetctl uninstall
+```
+
+### Help
+
+```text
+$ make
```

Comment:
These make targets seem like they're for the qleetctl repo.  We don't even have a Makefile in this repo yet :/

Reply:
i think you copied README from qleetctrl repo :)

Reply:
Who did?  You might want to check `git blame`